tools/config/go-templates/client.mustache

```diff
@@ -652,9 +652,16 @@ func (e *GenericOpenAPIError) SetError(errorString string) {
 	e.error = errorString;
 }
 
-// format error message using title and detail when model implements Error
-func formatErrorMessage(status, path, method string, v ApiError) string {
-	return fmt.Sprintf("%v %v: HTTP %v (Error code: %q) Detail: %v Reason: %v. Params: %v",
-			method, path, status, v.GetErrorCode(),
-	 		v.GetDetail(), v.GetReason(), v.GetParameters())
+// FormatErrorMessageWithDetails: format error message using error struct fields: Should be only used for testing
+func FormatErrorMessageWithDetails(status, path, method string, v ApiError) string {
```

Comment:
i guess my suggestion for func comment should be changed here in the template

Reply:
Yes. I always accept comment in 1 place and regenerate locally. 
Normal approach for SDK.